### PR TITLE
feat: Configure storybook theme

### PR DESCRIPTION
**Pull request by ArthurTriis1**
Signed-off-by: Arthur Andrade <arthurfelandrade@gmail.com>

## What's the purpose of this pull request?
This PR change theme of storybook to colors of VTex, add Store-ui logo, and do a sync of CHANGELOG.md file with storybook CHANGELOG page document.

## How it works? 
Screen Shot 2021-09-30 at 14 56 16

Screen Shot 2021-09-30 at 14 56 41

### CODE REVIEW

**Comment by igorbrasileiro** [comment]
@ArthurTriis1, can we change the color for that flags, one that's beside the bullets, for example.

File: packages/store-ui/.storybook/stories/releases/v0.x.x/changelog-template.stories.mdx [hunk 1]
```diff
@@ -1,25 +1,8 @@
+import { Description } from '@storybook/addon-docs/blocks'
 import { Meta } from '@storybook/addon-docs'
+import readme from '../../../../CHANGELOG.md'
+import { version } from '../../../../package.json'
 
-<Meta title="Releases/v0.x.x/CHANGELOG - template" />
+<Meta title={`Releases/Changelog`} />
```
Comment (emersonlaurentino) [inline packages/store-ui/.storybook/stories/releases/v0.x.x/changelog-template.stories.mdx:6]: As the title value is a simple string, you don't need to use it with back (`)
```suggestion
<Meta title="Releases/Changelog" />
```

File: packages/store-ui/.storybook/stories/releases/v0.x.x/changelog-template.stories.mdx [hunk 2]
```diff
@@ -1,25 +1,8 @@
+import { Description } from '@storybook/addon-docs/blocks'
 import { Meta } from '@storybook/addon-docs'
+import readme from '../../../../CHANGELOG.md'
```
Comment (emersonlaurentino) [inline packages/store-ui/.storybook/stories/releases/v0.x.x/changelog-template.stories.mdx:3]: Use a specific name, readme is not a good name for that.

```suggestion
import Changelog from '../../../../CHANGELOG.md'
```

File: packages/store-ui/.storybook/stories/releases/v0.x.x/changelog-template.stories.mdx [hunk 3]
```diff
@@ -1,25 +1,8 @@
...
-## 🐛 Fixes
-
-- [Button](?path=/story/atoms-button--button): removed `as` and `href` prop.
-
-## 🧹 Chores:
-
-- Created Releases section.
-- Created Migration Guide and CHANGELOG templates in Releases section.
-- Storybook upgraded to 6x and migration of our docs to MDX
+<Description>{readme}</Description>
```
Comment (emersonlaurentino) [inline packages/store-ui/.storybook/stories/releases/v0.x.x/changelog-template.stories.mdx:8]: Once we name the importing, we need to change it here too.
```suggestion
<Description>{Changelog}</Description>
```

**Review by emersonlaurentino — CHANGES_REQUESTED** [review]

**Review by tlgimenes — CHANGES_REQUESTED** [review]
Nice! Can you just make the logo smaller so it fits better on the header? thanks!

**Review by igorbrasileiro — APPROVED** [review]
Apply the suggestions left, and it's OK! Great job, @ArthurTriis1

**Comment by ArthurTriis1** [comment]
> @ArthurTriis1, can we change the color for that flags, one that's beside the bullets, for example?

I not found in [Theming Story UI documentation](https://storybook.js.org/docs/react/configure/theming) how do this, but i think that is possible, but not so trivial

**Review by emersonlaurentino — APPROVED** [review]
LGTM

**Pull request merged**